styles/css/vtex.product-price.css

```diff
@@ -37,7 +38,8 @@
 }
 
 .savings-discount--summary {
```

Comment:
Is this class being applied in any element?

Reply:
Found it, sry

Reply:
Yes! 
![image](https://user-images.githubusercontent.com/8443580/78823354-eddc5180-79b2-11ea-8750-6d21c76adb50.png)